src/routes/AccountCreationSteps.svelte

```diff
@@ -0,0 +1,107 @@
+<script>
+  import I18n from "i18n-js";
+  import eduIdLogo from "../img/logo_eduid.svg";
+  import arrowRightActive from "../icons/arrow-right-active.svg";
+  import arrowRightInactive from "../icons/arrow-right-inactive.svg";
+  import stepChecked from "../icons/step-checked.svg";
+  import stepInactive2 from "../icons/step-inactive-2.svg";
+  import stepInactive3 from "../icons/step-inactive-3.svg";
+  import stepActive1 from "../icons/step-active-1.svg";
+  import stepActive2 from "../icons/step-active-2.svg";
+  import stepActive3 from "../icons/step-active-3.svg";
+  import badge from "../icons/badge.svg";
+  import stepIcon2Active from "../icons/step-icon-building-active.svg";
+  import stepIcon2Inactive from "../icons/step-icon-building-inactive.svg";
```

Comment:
We can make this a bit smaller by adding an `index.js` file to the `icons` folder, and export every icon from there (like this: `export { default as stepInactive2 } from "./step-inactive-2.svg";`). Then we can do `import { stepInactive2, ... } from 'path/to/icons';` from wherever we need them.

Reply:
Done.